src/lib/components/alerts/emailVerificationBanner.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+    import { base } from '$app/paths';
+    import { goto } from '$app/navigation';
+    import { Button } from '$lib/elements/forms';
+    import { HeaderAlert } from '$lib/layout';
+    import { Typography } from '@appwrite.io/pink-svelte';
+    import { hideNotification, shouldShowNotification } from '$lib/helpers/notifications';
+    import { user } from '$lib/stores/user';
+    import { wizard } from '$lib/stores/wizard';
+    import { page } from '$app/state';
+
+    const { emailBannerClosed, onEmailBannerClose } = $props<{
+        emailBannerClosed: boolean;
+        onEmailBannerClose: (closed: boolean) => void;
+    }>();
+
+    const isOnOnboarding = $derived(() => page.route?.id?.includes('/(console)/onboarding'));
+
+    const hasUser = $derived(!!$user);
+    const needsEmailVerification = $derived($user && !$user.emailVerification);
+    const shouldShowNotificationBanner = $derived.by(() =>
+        shouldShowNotification('email-verification-banner')
+    );
+    const wizardNotActive = $derived(!$wizard.show && !$wizard.cover);
+    const bannerNotClosed = $derived(!emailBannerClosed);
+    const notOnOnboarding = $derived(!isOnOnboarding);
+
+    const shouldShowEmailBanner = $derived(
+        hasUser &&
+            needsEmailVerification &&
+            shouldShowNotificationBanner &&
+            wizardNotActive &&
+            bannerNotClosed &&
+            notOnOnboarding
+    );
+
+    function navigateToAccount() {
+        goto(`${base}/account`);
+    }
+
+    function handleDismiss() {
+        onEmailBannerClose(true);
+        hideNotification('email-verification-banner', { coolOffPeriod: 24 * 365 * 100 });
+    }
+</script>
+
+{#if shouldShowEmailBanner}
+    <HeaderAlert
+        type="warning"
+        title="Your email address needs to be verified"
```

Comment:
Copy here feels wrong 🤔 We say `needs to be verified` but in description we say `verification will be required soon`.

I think we need better title that more clearly indicates you just need to make sure you have access to your email, but no other action needed yet.